plugins/webhooks/server/tasks/DeliverWebhookTask.ts

```diff
@@ -423,12 +424,18 @@ export default class DeliverWebhookTask extends BaseTask<Props> {
       paranoid: false,
     });
 
+    const collection = model && (await presentCollection(undefined, model));
+    if (collection) {
+      // For backward compatibility, set a default color.
+      collection.color = collection.color ?? colorPalette[0];
```

Comment:
Perhaps we just change `color` to a getter on the server model and handle the fallback there once?

Reply:
The frontend now owns the `color` logic and setting a default color, if necessary; there's just too many state transitions to keep track of in the icon picker.
So the icon picker would send the final icon & color (null for an emoji) to the backend.

We're setting a default color here to ensure the user's webhooks do not fail since we changed the model's color field to accept `null` values now.